spec/inquisition/rubocop/categorizer_spec.rb

```diff
@@ -0,0 +1,40 @@
+RSpec.describe Inquisition::Rubocop::Categorizer do
+  describe '#find_category' do
+    subject { described_class.find_category(cop_name) == category }
+
+    context 'when offence category is security' do
```

Comment:
There should be 8 tests to cover all cases:
- when cop_name `Bundler/DuplicatedGem`
- when cop_name `Bundler/InsecureProtocolSource`
- when cop_name `Metrics/LineLength`
- when cop_name `Migrations/RemoveIndex`
- when cop_name namespace `Security`
- when cop_name namespace `Metrics`
- when cop_name namespace `Performance`
- when cop_name else